d2js/js/README.md

```diff
@@ -58,15 +60,39 @@ const result = await d2.compile('x -> y', {
 const svg = await d2.render(result.diagram, result.renderOptions);
 ```
 
+### Supporting Imports
+
+In order to support [imports](https://d2lang.com/tour/imports), a 'filesystem' needs to be passed to the D2 compiler. This is a mapping of D2 file paths to their content.
```

Comment:
Just `### Imports` followed by the example please. A filesystem needs to be passed regardless of whether using imports or not.